packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -3873,3 +3993,52 @@ class _CopySelectionAction extends ContextAction<CopySelectionTextIntent> {
   @override
   bool get isActionEnabled => state._value.selection.isValid && !state._value.selection.isCollapsed;
 }
+
+/// A class to return data for the `onContentCommitted` callback on text fields,
+/// which is when image content is inserted into the text field. The class holds
+/// information for the mime type, URI (location), and bytedata for the inserted
+/// content.
+@immutable
+class CommittedContent {
+  /// Creates a [CommittedContent] class. Any parameters can be null.
```

Comment:
Also split this into two paragraphs per https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#dartdoc-specific-requirements.